llama_index/vector_stores/pinecone.py

```diff
@@ -185,6 +188,7 @@ class PineconeVectorStore(BasePydanticVectorStore):
 
     _pinecone_index: Any = PrivateAttr()
     _tokenizer: Optional[Callable] = PrivateAttr()
+    _default_empty_query_vector: Optional[List[float]] = PrivateAttr()
```

Comment:
Does it make more sense to pass this in at the top-level, so that it can work with any vector store?

i.e. create a `QueryBundle` object in the auto-retriever that sets the query embedding vector to be blank zeros (which will then get used by the underlying vector db)

Reply:
hm. yes that's a good point. then not vector db specific